### ISSUE TITLE
Ix'aern (DRK)

### ISSUE THREAD

**Issue by benfolka** [LegionDark/Issues#739]
**Date & Time**:  07.20 (GMT +2)

**FFXI Client Version (use `/ver`)**: 30160203_0

**Character Name**: Clay

**Nation**:  Sandy

**Job(level)/Sub Job(level)**: 99 war/dnc

**NPC or Monster or item Name**: Ix'aern (DRK)

**Zone name & Coordinates (use `@where`)**:  -560 5 240

**ffxiah.com link (for items issues only)**: 

**Multi-boxing? (multiple clients on same connection)**:  no

**Steps To Reproduce / any other info**: Traded 48 to (f-10) aern organs to spawn point. it says "cant spawn mob are you sure it is already up"

**Comment by TeoTwawki** [comment]
3 guesses what the one and only condition for it saying that would actually be.

**Comment by benfolka** [comment]
i am wide scanning it is not up it is a force spawn.

**Comment by TeoTwawki** [comment]
the script literately checks for the state ix and its 2 helpers are in. You nto seeing it on widescan just means its not in your widescan view, which especially in that area isn't going to mean a heck of a lot.

**Comment by TeoTwawki** [comment]
Its not the script: its someone caused it to not load. its failing the check because the server thinks no such mob exists. I will change the check so this condition will not trigger that message again, and find how/why the server thinks it doesn't exists (we didn't change it, and it was working fine) 

I was trying to debug with you in game bug you either dc'd or logged just after I moved to the pop site.

**Comment by benfolka** [comment]
i may have caused it i traded 1 aern organ like a few days ago

**Comment by benfolka** [comment]
i have been there like forever and Ix'aern (DRG) did not spawn
not sure about this since it is a normal pop.

**Comment by TeoTwawki** [comment]
I had personally tested it when I added that message, he -was- working before the recent update. Nothing you could do as a player would cause the server to claim a mob with that ID didn't exist. This is an update bug. Somebodies custom sql line screwed up.

**Comment by benfolka** [comment]
i know it is too much to ask but could you check Ix'aern (DRG) ID too ?
sorry about that.

**Comment by TeoTwawki** [comment]
next time I'm in game I'll try

**Comment by TeoTwawki** [comment]
@R3P0FFXI 
tl;dr
mob group issues

**Comment by alannor** [comment]
Was able to pop DRG and MNK

**Comment by TeoTwawki** [comment]
Ix'Aern(DRK) workaround will be functioning again next time the map server restarts.
